Tests/OpenCombineTests/PublisherTests/MapErrorTests.swift

```diff
@@ -0,0 +1,257 @@
+//
+//  MapErrorTests.swift
+//  
+//
+//  Created by Joseph Spadafora on 7/4/19.
+//
+
+import XCTest
+
+#if OPENCOMBINE_COMPATIBILITY_TEST
+import Combine
+#else
+import OpenCombine
+#endif
+
+@available(macOS 10.15, *)
+final class MapErrorTests: XCTestCase {
+    static let allTests = [
+        ("testEmpty", testEmpty),
+        ("testError", testError),
+        ("testRange", testRange),
+        ("testNoDemand", testNoDemand),
+        ("testDemandSubscribe", testDemandSubscribe),
+        ("testDemandSend", testDemandSend),
+        ("testCompletion", testCompletion),
+        ("testCancel", testCancel),
+        ("testCancelAlreadyCancelled", testCancelAlreadyCancelled),
+        ("testLifecycle", testLifecycle),
+    ]
+
+    func testEmpty() {
+        // Given
+        let tracking = TrackingSubscriberBase<Int, OtherError>(
+            receiveSubscription: { $0.request(.unlimited) }
+        )
+        let publisher = PassthroughSubject<Int, TestingError>()
+        // When
+        publisher.mapError(OtherError.init).subscribe(tracking)
+        // Then
+        XCTAssertEqual(tracking.history, [.subscription(Subscriptions.empty)])
+    }
+
+    func testError() {
+        // Given
+        let expectedError = TestingError.oops
+        let tracking = TrackingSubscriberBase<Int, OtherError>(
+            receiveSubscription: { $0.request(.unlimited) }
+        )
+        let publisher = PassthroughSubject<Int, TestingError>()
```

Comment:
Let's use `CustomPublisher` instead of `PassthroughSubject` and see if this test will still pass